client/src/App.tsx

```diff
@@ -94,7 +94,7 @@ export const App = () => {
           </div>
           <Link to="/" onClick={() => {}}>
             <div id="IQEngineLogo" className="absolute mt-4 pt-2 top-0 left-1/2 transform -translate-x-1/2 flex">
-              <Feature flag="displayInternalBranding">
+              <Feature flag="hideInternalBranding" invert="true">
```

Comment:
I just fid this all too confusing to be honest reading this line I have no idea what will happen if I put hideInternalBranding as true will this hide the internal brand or display it because the invert is true ? I think would make more sense to add something like default and the default value for default would be true but you could set here a different value for default.

THinking about it would be much better to have a config file like the constants that we do have that would have the feature flags that are supported with the default value for them and them we would use that to set defaults instead of setting them here. the good thing of that approach it si that we aocul also use that to validate the allowed string values for feature flags to avoid misspelling and things like that.